### ISSUE TITLE
Option to disable plots window

### ISSUE THREAD

**Issue by abudden** [spyder-ide/spyder#15666]
### Issue Report Checklist

* [X] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [X] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [X] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* N/A Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* N/A Tried basic troubleshooting (if a bug/error)
* [X] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

I would like to be able to disable the plots window, but I can't find an option to allow me to do this.

I use the Variable Explorer extensively, and every time I start Spyder, the first thing I do is switch from the Help tab (which I never use) to the Variable Explorer tab.  I have the Graphics Backend setting set to Automatic, which causes matplotlib plots to appear in another window, with all the (wonderful) interactive tools available.  That all works really well (although defaulting to the Variable Explorer tab instead of Help would be a nice touch).

When I use sympy, I use this:

```python
import sympy as sy
from IPython.display import display
sy.init_printing()
```

I can then use `display` to print `sympy` equations (assuming I've disabled the "Mute inline plotting" option).  However, when I display an equation in the console, it also puts it into the plots window and switches the display from Variable Explorer to the Plots window.  I have to manually remove all plots from the plots window to get rid of the equations and then manually switch back to the Variable Explorer.

If I just type the name of the variable into the console, it works as I would like, but I can't make this work within a script - hence the use of IPython.display, which sort of works but has the undesired side-effects with the plots window.

### What steps reproduce the problem?

1. Disable the "Mute inline plotting" option
2. Set graphics backend to "Automatic"
3. Verify that matplotlib figures open in a new windows as desired
4. Make sure the Variable Explorer is visible
5. Create a script that creates a sympy equation and uses IPython.display to print it to the console and run it:

```python
import sympy as sy
from IPython.display import display
sy.init_printing()

s = sy.Symbol('s')
g = 1 / sy.Poly([1, 10], s)
display(g)
```

### What is the expected output? What do you see instead?

What should happen: Equation should be latex-ified and printed to the console; Variable Explorer should stay visible; Plots window should stay empty.

What actually happens: Equation is latex-ified and printed to the console (correct), Plots window becomes visible, hiding Variable Explorer (incorrect), Plots window gains a new "Plot" consisting of the image of the latex-ified equation (incorrect)

## Versions

I've tried `conda update spyder`, but the more recent 5.0.x versions don't seem to be available via conda yet.

* Spyder version: 5.0.0 None
* Python version: 3.7.10 64-bit
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  22.3.0 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.8.1 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                :  2.3.1 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  20.0.0 (OK)

# Optional:
cython >=0.21                 :  0.29.23 (OK)
matplotlib >=2.0.0            :  3.3.4 (OK)
numpy >=1.7                   :  1.20.1 (OK)
pandas >=1.1.1                :  1.2.4 (OK)
scipy >=0.17.0                :  1.6.2 (OK)
sympy >=0.7.3                 :  1.8 (OK)
```

**Comment by juanis2112** [comment]
@abudden thanks for suggesting this. I think this is a good idea. @ccordoba12 is there an easy way to implement this so that we can add an option (I guess in preferences) to disable the plots pane?

**Comment by ccordoba12** [comment]
Hey @abudden and @juanis2112, I fixed this issue in our 5.0.2 version, released a couple of weeks ago. Let's hope it'll be added to Anaconda soon.

If you disable "Mute inline plotting" and hide the Plots pane, it won't be shown again.
